packages/application-shell/src/test-utils/test-utils.js

```diff
@@ -285,6 +292,15 @@ const experimentalRender = (ui, renderOptions) => {
   });
 };
 
+const defaults = {
+  mocks: {
+    user: defaultUser,
+    project: defaultProject,
+    environment: defaultEnvironment,
+    permissions: defaultPermissions,
+  },
+};
```

Comment:
Just an idea. It makes the tests a bit simpler. We could also expose the default rtl render there. /cc @emmenko

Reply:
> Just an idea. It makes the tests a bit simpler.

What do you mean? Those values are already available as

```js
const { user, project, environment } = renderApp(<Foo />);
```

> We could also expose the default rtl render there

What do you mean? It's already exported with `export * from 'react-testing-library';`

Reply:
I know. It would just be easier to to test the test-utils. Anyway. Will remove.